### PR TITLE
Antiquarian: create and remove logic

### PR DESCRIPTION
**Pull request by fobadara**
In this branch I:

✏️  setup Redux store

✏️ created the necessary actions and reducer functions

### CODE REVIEW

File: src/components/addbook.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <>
+      <h2>Add New Book</h2>
+      <form action="#">
+        <input type="text" required />
+        <button type="submit" onClick={submitBookToStore}>Add Book</button>
+      </form>
+    </>
+  );
```
Comment (dasileker) [inline src/components/addbook.js]: - [ ] The objective of the redux store is to take the data in input and add it to the store, right now I can only see the `submitBookToStore => object` are added.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @fobadara,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/addbook.js (hunk 1 above)
Comment (fobadara) [reply src/components/addbook.js]: Hello @dasileker,
Thanks for the review. I can see what you pointed out and I have adjusted my code. But from what I can understand from the instructions this isn't a requirement at this stage.

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (ZeenatLawal) [inline public/manifest.json:1]: - [OPTIONAL] All boilerplate files and icons should be deleted as they are not required for the project. e.g In the public folder(the logo images, manifest.json and robots.txt) and in the src folder(logo.svg, reportwebvitals).

**Review by ZeenatLawal — APPROVED** [review]
Hi @fobadara, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: public/manifest.json (hunk 2 above)
Comment (fobadara) [reply public/manifest.json:1]: Thanks, @ZeenatLawal. I will take note of your suggestion on my next pull request.

**Pull request merged**